### ISSUE TITLE
Feature: Manage pending Expenses and Charges

### ISSUE THREAD

**Issue by kewitz** [opencollective/opencollective#5431]
## User story 
As a user that receives an invite to submit an expense, I want to be able to see that expense draft on the platform so I can submit that expense without having to open my email.

As a user that has a recurring expense set up, I want to be able to see pending drafts on the platform so I can submit a new expense without having to open my email.

As an admin of a collective that has a virtual card, I want to be able to see charge expenses missing receipts on the platform so I can quickly submit a new receipt without having to open my email.

## Best solution for this problem
Create a page where I can list all the expenses that are waiting on my action to be approved/paid.

It should also list the expenses that belong to any collective I'm a member of.

## MVP
A page like the one we use to list all pending member invitations could solve the issue.

**Comment by BenJam** [comment]
could we just add pending/draft status to `https://opencollective.com/user/transactions` ?

**Comment by kaylarep** [comment]
not sure if this is the right place for this feedback from [a collective's 'About' section](https://opencollective.com/ward3neighborsupport):
![image](https://user-images.githubusercontent.com/64920686/164339641-cea22b5b-5fc1-4f94-ba4c-3b37b1175459.png)

**Comment by BenJam** [comment]
okay i still think the way to solve this is to add a section/filter/details to the `/user/transactions` page. What do you think @kewitz ?

![Screenshot 2022-06-30 at 14 53 1](https://user-images.githubusercontent.com/158833/176696949-bded0bfc-8973-4b5a-aae1-d6967ea5938d.png)

**Comment by kewitz** [comment]
@BenJam the problem is that transactions are created only after the expense is settled.
In the case of credit card charges, we would only be able to list the transaction on the Collective Transactions page, which could also go unnoticed.

**Comment by Betree** [comment]
> As a user that has a recurring expense set up, I want to be able to see pending drafts on the platform so I can submit a new expense without having to open my email.

This seems to be possible already, I can see pending drafts for my recurring expenses on https://opencollective.com/engineering/expenses. But I'm not able to edit them directly: https://github.com/opencollective/opencollective/issues/5720

**Comment by RichardLitt** [comment]
> As an admin of a collective that has a virtual card, I want to be able to see charge expenses missing receipts on the platform so I can quickly submit a new receipt without having to open my email.

I would dearly like this. Right now, there is no way for me to know that an expense is lacking receipts through the platform. As this cancels my cards over time, it would be so useful to be able to see this so I can solve the issue before I have to go through and manually find it by clicking all of the expenses or by digging through my emails.

**Comment by RichardLitt** [comment]
Related to https://github.com/opencollective/opencollective/issues/4312.

**Comment by Betree** [comment]
Part of this issue will be addressed by https://github.com/opencollective/opencollective-frontend/pull/8381:

> As a user that has a recurring expense set up, I want to be able to see pending drafts on the platform so I can submit a new expense without having to open my email.

By going to User Menu > Submitted expenses, you can filter to get all your drafts:

![image](https://user-images.githubusercontent.com/1556356/202709136-72f29474-ee53-49f9-91f7-92642c4e1680.png)

![image](https://user-images.githubusercontent.com/1556356/202709242-c05339bc-2b1f-46c6-a693-26170944f575.png)

**Comment by BenJam** [comment]
I think #3749 fixes this, closing.

**Comment by RichardLitt** [comment]
I'm not seeing draft as an option, nor "Submitted Expenses" in the User dropdown. 

<img width="1362" alt="Screenshot 2023-01-09 at 20 40 34" src="https://user-images.githubusercontent.com/910753/211442468-0bef9fc0-f2d2-46ff-9944-cd8ccf0cb380.png">

**Comment by Betree** [comment]
> I'm not seeing draft as an option

@RichardLitt This is not something we've implemented, but it has been discussed during demos. I've documented that in https://github.com/opencollective/opencollective/issues/6300.

> nor "Submitted Expenses" in the User dropdown.

The link has been renamed to "Manage expense" after getting feedback from the team in this direction.

![image](https://user-images.githubusercontent.com/1556356/211789860-c0da7518-0a85-4c5b-8a28-e581b52a1361.png)
